sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -2649,3 +2652,84 @@ case class Sentences(
     copy(str = newFirst, language = newSecond, country = newThird)
 
 }
+
+case class ToPrettyString(expr: Expression, timeZoneId: Option[String] = None)
+  extends UnaryExpression with ImplicitCastInputTypes with TimeZoneAwareExpression{
+  import ToPrettyString._
+
+  require(children.nonEmpty, s"$prettyName() should take at least 1 argument")
+  override def withTimeZone(timeZoneId: String): TimeZoneAwareExpression =
+    copy(timeZoneId = Option(timeZoneId))
+  override def child: Expression = expr
+  override def foldable: Boolean = child.foldable
+  override def nullable: Boolean = child.nullable
+  override def dataType: DataType = StringType
+  override def inputTypes: Seq[AbstractDataType] = Seq(AnyDataType)
```

Comment:
let's not implement `ImplicitCastInputTypes` if there is no type checking.

Reply:
Got it, we don't need check input type for this expression